Lib/gftools/builder/__init__.py

```diff
@@ -281,7 +285,22 @@ def run_fontmake(self, source, args):
             original_output_dir = args["output_dir"]
             tmpdir = tempfile.TemporaryDirectory()
             args["output_dir"] = tmpdir.name
-        args["flatten_components"] = self.config["flattenComponents"]
+
+        if (
+            self.config["flattenComponents"] or
+            self.config["decomposeTransformedComponents"]
+        ):
+            filters = args.get("filters", [])
+            if self.config["flattenComponents"]:
+                filters.append([
+                    loadFilterFromString("FlattenComponentsFilter")
+                ])
+
+            if self.config["decomposeTransformedComponents"]:
+                filters.append([
+                    loadFilterFromString("DecomposeTransformedComponentsFilter")
+                ])
```

Comment:
```suggestion
                filters.append(
                    loadFilterFromString("DecomposeTransformedComponentsFilter")
                )
```

Same here :-)

Reply:
Currently, ufo2ft doesn't like the lists being appended. If I remove them, then it's fine.

Reply:
Here's a minimal test case

[component_test.zip](https://github.com/googlefonts/gftools/files/6584956/component_test.zip)

Reply:
Thanks! It's fixed.